sdk/storage/Azure.Storage.Files.DataLake/perf/Azure.Storage.Files.DataLake.Perf/Scenarios/Upload.cs

```diff
@@ -0,0 +1,134 @@
+﻿//Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.Storage.Files.DataLake.Perf.Scenarios
+{
+    /// <summary>
+    ///   The performance test scenario focused on uploading to Data Lake storage.
+    /// </summary>
+    ///
+    /// <seealso cref="Azure.Test.Perf.PerfTest{SizeOptions}" />
+    ///
+    public sealed class Upload : PerfTest<SizeOptions>
+    {
+        /// <summary>
+        ///   The ambient test environment associated with the current execution.
+        /// </summary>
+        ///
+        private static PerfTestEnvironment TestEnvironment { get; } = PerfTestEnvironment.Instance;
+
+        /// <summary>
+        ///   The name of the file system to use across parallel executions of the scenario.
+        /// </summary>
+        ///
+        private static string FileSystemName { get; } = Guid.NewGuid().ToString();
+
+        /// <summary>
+        ///   The client for interaction with the Data Lake file system.
+        /// </summary>
+        ///
+        private DataLakeFileSystemClient FileSystemClient { get; }
+
+        /// <summary>
+        ///   The payload to use with a file being uploaded.
+        /// </summary>
+        ///
+        private Stream Payload { get; set; }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="Read"/> class.
```

Comment:
Typo - 'Upload'

Reply:
I like how you're being polite and calling it a typo rather than saying "learn how to cut and paste, n00b!"   😄